src/Core/src/Primitives/FontWeight.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+namespace Microsoft.Maui
+{
+	public enum FontWeight
+	{
+		Regular = 400,
```

Comment:
Really like these numbers. Technically (like SkiaSharp) you can pass in any int value and then the underlying system can handle that. For example, some person may make a font of 450 :)

Reply:
All the platforms but iOS can handle any arbitrary integer!